Zend/zend.h

```diff
@@ -107,6 +107,28 @@ typedef struct _zend_trait_alias {
 	uint32_t modifiers;
 } zend_trait_alias;
 
+typedef struct _zend_class_muttable_data {
```

Comment:
Just a typo: muttable instead of mutable?

Reply:
And in a bunch of other places.